### ISSUE TITLE
How to get mmdetection ssd300_coco_int8 quantized model?

### ISSUE THREAD

**Issue by ksmultiacc3** [openvinotoolkit/nncf_pytorch#154]
I followed the [branch](https://github.com/aleksei-kashapov/nncf_pytorch/tree/update_mmdet_patch) and was running the ssd300_coco_int8 quantization aware training. I wanted to know how I can get the int8 models. I ran
`python tools/train.py configs/nncf_compression/ssd/ssd300_coco_int8.py`
and it creates a output folder inside which there are .pth files. But when I load these, it contains weights of type torch.cuda.FloatTensor which is 32 bit floating point. Please tell how I can get the (torch.int8) int8 model weights.

**Comment by vshampor** [comment]
Greetings, @ksmultiacc3!

NNCF only performs a simulation of actual INT8 quantization that would be executed on target hardware. All calculations in NNCF are done in FP32, but the compressed model acts identically to the presumed low-bitwidth or sparse execution environments in terms of quantization of operation weights and input tensors. This simulation is enough to perform quantization-aware training of model parameters and thus improve endpoint accuracy during low-bitwidth execution on actual hardware.

The compressed model itself can nevertheless be exported to the ONNX format supported by the OpenVINO runtime to be executed in actual INT8 on Intel hardware and utilize the corresponding performance improvements. Exporting the NNCF-compressed INT8 models for running INT8 inference using PyTorch runtime and its `torch.int8` data types, however is currently unsupported. Please check out Intel's [OpenVINO](https://github.com/openvinotoolkit/openvino) offering to determine whether it could fit your DNN inferencing purposes instead.
